### PR TITLE
PS-7394 Select correct VERSION file

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

**Comment by inikep** [comment]
We have the same issue with 5.7 e.g. https://ps57.cd.percona.com/job/percona-server-5.7-param/348/

File: build-ps/version.sh [hunk 1]
```diff
@@ -3,7 +3,14 @@ DIR=$(dirname $(realpath $0))
 BRANCH=$(git branch --show-current)
 IFS='.' read -r MAJOR MINOR PATCH <<< $(echo $BRANCH | awk -F'-' '{print $2}')
 EXTRA=$(echo $BRANCH | awk -F'-' '{print $3}')
-source ${DIR}/../VERSION
+if [ -f VERSION ]; then
```
Comment (vorsel) [inline build-ps/version.sh]: Shouldn't we check ${DIR}/../VERSION instead of VERSION ?

File: build-ps/version.sh [hunk 2]
```diff
@@ -3,7 +3,14 @@ DIR=$(dirname $(realpath $0))
...
+    source ${DIR}/../VERSION
+elif [ -f MYSQL_VERSION ]; then
+    source ${DIR}/../MYSQL_VERSION
```
Comment (vorsel) [inline build-ps/version.sh:9]: The same as above.

File: build-ps/version.sh (hunk 1 above)
Comment (EvgeniyPatlan) [reply build-ps/version.sh]: fixed

File: build-ps/version.sh (hunk 2 above)
Comment (EvgeniyPatlan) [reply build-ps/version.sh:9]: fixed

**Review by vorsel — APPROVED** [review]

**Pull request merged**